snapd.go

```diff
@@ -523,6 +524,14 @@ func action(ctx *cli.Context) {
 							}).Error(err)
 						}
 					}
+					if err != nil {
```

Comment:
You are most probably right however the code your are referring to is not part of my pull request changes ;-)